<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Volume</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="volumeSlider">
      <property name="toolTip">
       <string>Volume Control</string>
      </property>
      <property name="toolTipDuration">
       <number>2</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="accessibleName">
       <string>Tab</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="mediaTab">
       <attribute name="title">
        <string> Media Player</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="2">
         <widget class="QPushButton" name="nextSongBtn">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QListWidget" name="songList"/>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="playPauseBtn">
          <property name="accessibleName">
           <string>play</string>
          </property>
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Songs</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="prevSongBtn">
          <property name="text">
           <string>Previous</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="radioTab">
       <attribute name="title">
        <string> Radio</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Playlist</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="serverIp">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Server IP</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="QListWidget" name="playlistList">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="connectServerBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="disconnectServerBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="sendFileBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Send File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dwldFileBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Download Selected</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="voiceTab">
       <attribute name="title">
        <string> Voice Chat</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="2">
         <widget class="QPushButton" name="connectPeerVoiceBtn">
          <property name="accessibleName">
           <string>connect</string>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="peerVoicePort">
          <property name="inputMask">
           <string>9999</string>
          </property>
          <property name="text">
           <string>7000</string>
          </property>
          <property name="placeholderText">
           <string>Peer Port</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="recordBtn">
            <property name="text">
             <string>Record</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopRecordBtn">
            <property name="accessibleName">
             <string>stoptest</string>
            </property>
            <property name="text">
             <string>Stop Record</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="peerVoiceIp">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Peer IP</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>557</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>serverIp</tabstop>
  <tabstop>playlistList</tabstop>
  <tabstop>volumeSlider</tabstop>
  <tabstop>connectPeerVoiceBtn</tabstop>
  <tabstop>peerVoicePort</tabstop>
  <tabstop>peerVoiceIp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
